src/NServiceBus.Hosting.Windows/Roles/AsA_Client.cs

```diff
@@ -4,5 +4,10 @@ namespace NServiceBus
     /// Indicates this endpoint is a client.
     /// As such will be set up as a non-transactional endpoint with no impersonation and purging messages on startup.
     /// </summary>
+
+    [ObsoleteEx(
+    TreatAsErrorFromVersion = "7.1",
+    Message = "The functionality of AsA_Server, and AsA_Publisher has been made defaults in the core.",
```

Comment:
"The AsA_Server, and AsA_Publisher roles are obsoleted. Manually configure the EndpointConfiguration object via IConfigureThisEndpoint.Customize(EndpointConfiguration endpointConfiguration)"